src/components/ServiceDetails/ExistingModel/index.js

```diff
@@ -42,7 +47,9 @@ const ExistingModel = ({
     try {
       startMMconnectLoader();
       const sdk = await initSdk();
+      setSdkService(sdk);
       const address = await sdk.account.getAddress();
+      setAddressService(address);
```

Comment:
You don't need set the address over here remove that setAddressService state. You can directly access address using the wallet.address and it's accessible in all the files